packages/app/src/debug/DebugOverLimit.vue

```diff
@@ -27,15 +28,65 @@ import { useI18n } from '@cy/i18n'
 import { getUrlWithParams } from '@packages/frontend-shared/src/utils/getUrlWithParams'
 import { computed } from '@vue/reactivity'
 
+export type DataRetentionLimitExceeded = { __typename: 'DataRetentionLimitExceeded', dataRetentionDays: number | null }
+
+export type UsageLimitExceeded = | { __typename: 'UsageLimitExceeded', monthlyTests: number | null }
+
+export type CloudRunHidingReason = DataRetentionLimitExceeded | UsageLimitExceeded
```

Comment:
Can we create a fragment out of this and derive the types from it?

```
gql`
fragment DebugReasonsRunIsHidden on CloudRun {
  id
  reasonsRunIsHidden {
    __typename
    ... on DataRetentionLimitExceeded {
      dataRetentionDays
    }
    ... on UsageLimitExceeded {
      monthlyTests
    }
  }
}
`

type CLoudRunHidingReason = DebugReasonsRunIsHidden['reasonsRunIsHidden']
```

That way we don't have to duplicate types.